specification/indices/field_usage_stats/IndicesFieldUsageStatsRequest.ts

```diff
@@ -74,11 +69,5 @@ export interface Request extends RequestBase {
      * Comma-separated list or wildcard expressions of fields to include in the statistics.
      */
     fields?: Fields
-    /**
-     * The number of shard copies that must be active before proceeding with the operation.
-     * Set to all or any positive integer up to the total number of shards in the index (`number_of_replicas+1`).
-     * @server_default 1
-     */
-    wait_for_active_shards?: WaitForActiveShards
```

Comment:
This parameter is not read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestFieldUsageStatsAction.java#L41-L52 - I also checked that it was not accepted in practice.